Resources/Prototypes/_LostParadise/Datasets/dreamadjectives.yml

```
@@ -0,0 +1,959 @@
+﻿- type: dataset
+  id: dreamadjectives
+  values:
+  - новый
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)